src/test/java/com/epam/tc/hw7/JdiTest.java

```diff
@@ -43,5 +33,22 @@ public void loginTest() {
 
         //Open Metals & Colors page by Header menu
         JdiSite.userPage.openPageFromHeaderMenu(MenuOptions.METALS_AND_COLORS);
+
+        //Fill form Metals & Colors by data from json
+        JdiSite.metalsAndColorsPage.chooseSummary(dataFromJson.getSummary());
+        JdiSite.metalsAndColorsPage.chooseElements(dataFromJson.getElements());
+        JdiSite.metalsAndColorsPage.chooseColor(dataFromJson.getColor());
+        JdiSite.metalsAndColorsPage.chooseMetal(dataFromJson.getMetals());
+        JdiSite.metalsAndColorsPage.chooseVegetables(dataFromJson.getVegetables());
+
+        //Submit form Metals & Colors
+        JdiSite.metalsAndColorsPage.submitFormMetalsColors();
+
+        //Result sections should contains data from json
+        JdiSite.metalsAndColorsPage.checkResultLog(dataFromJson);
+
+        //Logout
+        JdiSite.userPage.logoutFromUserAccount();
```

Comment:
It's a bad practice to login and logout for each test run. Should login only once

Reply:
changed